### ISSUE TITLE
Invalid Signatures with 2.3.15

### ISSUE THREAD

**Issue by topik0** [Windscribe/Desktop-App#11]
I am given a "Can't start the engine" error when attempting to open Windscribe on Arch Linux.  I looked at ~/.local/share/Windscribe/Windscribe2/log_gui.txt and found the following output:
```
[231221 22:54:49:715      0.342] [basic]	 "Calling Engine at: /usr/local/windscribe/WindscribeEngine"
[231221 22:54:49:715      0.342] [basic]	 Verifying  "/usr/local/windscribe/WindscribeEngine"  with  "/usr/local/windscribe/signatures/WindscribeEngine.sig"  and  ":/keys/linux/key.pub"
[231221 22:54:49:735      0.361] [basic]	 Executable's signature does not match signature file
[231221 22:54:49:735      0.362] [basic]	 Engine signature invalid
```
This error is not present when running a binary installed via compiling from source, but is present when installing from the deb binary on the releases page.
Is this on your guys' end, or my end?  Thanks!

**Comment by bernerdad** [comment]
I used debtap to convert the 2.3.15 debian package to a zst, and installed via pacman.  The app starts up without any signature errors.  However, both wireguard and openvpn are experiencing connection issues that we'll have to diagnose when we start work on the official Arch Linux port after the Christmas break.  Cheers!

**Comment by H1llllll** [comment]
ping

**Comment by bernerdad** [comment]
Looks to be fixed as of v2.5.1, available in our guinea pig channel.
